### PR TITLE
Enable Abort Button in Add Validator and Register Validator

### PR DESCRIPTION
**Pull request by Esteban100k**
Enabled the Abort Button in the Add Validator and Register Validator transactions

### CODE REVIEW

**Review by aeddaqqa — CHANGES_REQUESTED** [review]
can you please use the abort signin modal you can find it in src/components/wallet/earn/ModalAbortSigning.vue so we will have unified abort functionality in all pending trxs

**Review by aeddaqqa — CHANGES_REQUESTED** [review]
- there is a duplication of the phrase : 
Screenshot 2023-05-12 at 20 37 33
Screenshot 2023-05-12 at 20 35 04

- also if i aborted the tx with the second user the reload button disappear and i need to logout and login again to check if the section is updated
- if i register a validator with the first user and the second is already logged the reload button does not work

**Review by aeddaqqa — DISMISSED** [review]

File: src/components/wallet/earn/Validate/PendingMultisig.vue [hunk 1]
```diff
@@ -346,6 +344,7 @@ export default class PendingMultisig extends Vue {
                     message: 'Transaction has been cancelled',
```
Comment (rafaelkordonias) [inline src/components/wallet/earn/Validate/PendingMultisig.vue]: Please use translations and not hardcoded text here

**Review by rafaelkordonias — CHANGES_REQUESTED** [review]

File: src/components/wallet/earn/Validate/PendingMultisig.vue (hunk 1 above)
Comment (Esteban100k) [reply src/components/wallet/earn/Validate/PendingMultisig.vue]: @rafaelkordonias Translations are now used

**Review by aeddaqqa — APPROVED** [review]

**Review by rafaelkordonias — APPROVED** [review]

**Pull request merged**